server/src/tram_track_graph_transformer/tram_track_graph_transformer.py

```diff
@@ -61,6 +67,11 @@ def _build_tram_track_graph_from_osm_ways(self):
 
         return graph
 
+    def _get_node_type(self, node: Node):
+        if node.id in self._city_configuration.custom_stop_mapping.values():
+            return NodeType.TRAM_STOP
+        return NodeType.get_by_value_safe(node.tags.get("railway"))
```

Comment:
A ternary operator wouldn't look to bad here:
```suggestion
        return (
            NodeType.TRAM_STOP
            if node.id in self._city_configuration.custom_stop_mapping.values()
            else NodeType.get_by_value_safe(node.tags.get("railway"))
        )
```